docs/site/Using-TypeORM-with-LoopBack.md

```diff
@@ -0,0 +1,132 @@
+---
+lang: en
+title: 'How to use TypeORM with LoopBack'
+keywords: LoopBack 4.0, LoopBack 4, TypeORM
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Using-TypeORM-with-LoopBack.html
+---
+
+## Overview
+
+If you would like to use TypeORM with LoopBack, it is possible with the use of
+`@loopback/typeorm`. Install it as a dependency in your project.
+
+```sh
+npm install --save @loopback/typeorm
+```
+
+Then, import `TypeOrmMixin` from `@loopback/typeorm` and apply it to your
+application class as shown below.
+
+```ts
+import {BootMixin} from '@loopback/boot';
+import {RestApplication} from '@loopback/rest';
+import {TypeOrmMixin} from '@loopback/typeorm';
+export class MyApplication extends BootMixin(TypeOrmMixin(RestApplication)) {
+  ...
+}
+```
+
+### Creating Entities
+
+[Entities](https://typeorm.io/#/entities) are equivalent to LoopBack's Models.
+Define the entities as usual and keep them in a directory named `entities`.
+
+```ts
+// src/entities/book.entity.ts
+import {Entity, Column, PrimaryColumn} from 'typeorm';
+@Entity()
+export class Photo {
+  @PrimaryColumn()
+  id: number;
+
+  @Column()
+  title: string;
+
+  @Column()
+  isPublished: boolean;
+}
+```
+
+### Creating Connections
```

Comment:
Maybe it's better to have `Creating Connections` come before `Creating Entities`?